src/Octoliner.cpp

```diff
@@ -283,21 +254,13 @@ void Octoliner::changeAddrWithUID(uint8_t newAddr) {
 
     delay(1);
```

Comment:
Do we need a delay here? If needed, please document this.

Reply:
This is not my code. I don't know exactly why Vasily and Morgan added these dalay.
From the very fact of the appearance of this function (changeAddrWithUID), I can assume that they encountered the instability of i2c at high speeds and they had to add crutches to somehow make it all work.
Perhaps there are also problems in the implementation of the i2c slave in the code for stm32f030.
In my opinion, only the third delay is needed - you need to give time to program a new value in FLASH.

Reply:
I also think that a delay might be needed when writing to Flash. Can we check it out? Or leave it on the "don't touch the code that works" principle?

Reply:
I'll check when I'm in the office, I'll need an oscilloscope.

Reply:
Do I understand correctly that we can accept this PR, and after checking it on an oscilloscope, we will release a patched version?

Reply:
look at my email please

Reply:
I did not get the mail.